### ISSUE TITLE
Cache Layer Error: cache.bucketName

### ISSUE THREAD

**Issue by mfurkanoztrk** [Canner/vulcan-sql#260]
## Environment

- Version:
cli version: 0.6.0 
core version: 0.6.0 
build version: 0.6.0 
serve version: 0.6.0 

- OS: macOS intel i7

## What's wrong?

I want to use cache layer but i got this error:

cache.bucketName in profile "bq" is required when using cache feature.

bq is my primary data source, that is i want to get data from bq and keep cache on duckdb. 
how do i set cache.bucketName in profile? there is no specification about this on documentation.

Thanks fro your support in advance.

**Comment by cyyeh** [comment]
@mfurkanoztrk 

You'll need to fill in the `profiles.yaml` and we'll update the documentation in the near future. In short, you need to add the `cache` and `bucketName` fields.

However, we found another issue and we'll fix it ASAP.

```yaml
- name: bq # profile name
  type: bq
  connection:
    # Location must match that of the dataset(s) referenced in the query.
    location: US
    # Optional: The max rows we should fetch once.
    chunkSize: 100
    # The project ID from the Google Developer's Console, e.g. 'grape-spaceship-123'. We will also check the environment variable `GCLOUD_PROJECT` for your project ID. If your app is running in an environment which [supports](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application) Application Default Credentials), your project ID will be detected.
    projectId: 'your-project-id'
    # Full path to the a .json, .pem, or .p12 key downloaded from the Google Developers Console. If you provide a path to a JSON file, the `projectId` option above is not necessary. NOTE: .pem and .p12 require you to specify the `email` option as well.
    keyFilename: '/path/to/keyfile.json'
  cache:
    # The GCS bucket name for vulcan to store query result that will be used in cache feature
    bucketName: 'your-bucket-name'
```

**Comment by cyyeh** [comment]
@mfurkanoztrk Hi! We've fixed the issue in the 0.8.0 release! Hope you enjoy using VulcanSQL! Thank you!
